misc/estimate_batch.test.js

```diff
@@ -0,0 +1,119 @@
+const { TezosToolkit, MichelsonMap, OpKind } = require("@taquito/taquito");
```

Comment:
the files in the `misc` folder are just a bunch of scripts I use for testing various aspects of the contracts. This is not production code, just test snippets